docs/quiz.css

```diff
@@ -1,97 +1,131 @@
 html, body {
-    height: 100%;
+  height: 100%;
 }
 
 .page {
-    width: 100%;
-    height: 100%;
-    border-collapse: collapse;
-    display: table;
+  width: 100%;
+  height: 100%;
+  border-collapse: collapse;
+  display: table;
 }
 
 .container {
-    max-width: 650px;
+  max-width: 650px;
 }
 
 .notbold {
-    font-weight: 100;
+  font-weight: 100;
 }
 
 .hljs {
-    background: transparent;
+  background: transparent;
 }
 
 #text-output {
-    display: inline;
-    margin-top: -100%;
-    width: 100px;
+  display: inline;
+  margin-top: -100%;
+  width: 100px;
 }
 
 .reset {
-    display: table-row;
-    vertical-align: bottom;
-    text-align: right;
-    height: 0;
-    line-height: 0;
+  display: table-row;
+  vertical-align: bottom;
+  text-align: right;
+  height: 0;
+  line-height: 0;
 }
 
 .reset-link {
-    display: block;
-    height: 0;
-    position: relative;
-    right: 10px;
-    bottom: 10px;
+  display: inline-block;
+  height: 0;
+  position: relative;
+  right: 10px;
+  bottom: 10px;
 }
 
 .reset-link:after {
-    content: "[Reset]";
+  content: "[Reset]";
 }
 
 #accordion {
-    transition: font-size .25s, opacity .5s .10s;
-    padding-left: 10px;
+  transition: font-size 0.25s, opacity 0.5s 0.1s;
+  padding-left: 10px;
 }
 
 @media screen and (min-width: 450px) {
-    #accordion {
-        padding-left: 30px !important;
-    }
+  #accordion {
+    padding-left: 30px !important;
+  }
 }
 
 #accordion:not(.show) {
-    font-size: 0;
-    opacity: 0;
-    transition: opacity .25s, font-size .5s .15s;
+  font-size: 0;
+  opacity: 0;
+  transition: opacity 0.25s, font-size 0.5s 0.15s;
```

Comment:
Why is this better, and if it's better, why is it not done consistently for other decimals, like the one you added in `pre code`?

Reply:
it's just a fe abbr